ballsdex/packages/balls/cog.py

```diff
@@ -696,3 +696,41 @@ async def count(
             f"You have {balls} {special_str}{shiny_str}"
             f"{country}{settings.collectible_name}{plural}{guild}."
         )
+
+    @app_commands.command()
+    async def dispose(self, interaction: discord.Interaction, ball: BallInstanceTransform):
+        """Dispose a countryball for coins.
+
+        Parameters
+        ----------
+        ball: BallInstance
+            The countryball you want to dispose of."""
+        if not ball:
+            return
+        if ball.countryball.enabled or not ball.is_tradeable:
+            await interaction.response.send_message(
+                f"You cannot dispose of this {settings.collectible_name}.", ephemeral=True
+            )
+            return
+        await interaction.response.defer()
+        coins = ball.countryball.coin_amount // 2
+        view = ConfirmChoiceView(interaction)
+        await interaction.response.send_message(
```

Comment:
```suggestion
        await interaction.followup.send(
```
the command wouldn't work otherwise